draft-ietf-rats-corim.md

```diff
@@ -1151,7 +1156,8 @@ The new entry is added to the existing set of entries using the Endorser's autho
 
 #### Conditional Endorsement Triple {#sec-comid-triple-cond-endors}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/310
+A Conditional Endorsement Triple declares one or more conditions that, once they match, cause every entry in the endorsements to be added to the accepted state.
```

Comment:
have we already defined "accepted state" at this point?  If not, I suggest using a more intelligible term.